exec/java-exec/src/main/java/org/apache/drill/exec/planner/RuleInstance.java

```diff
@@ -140,4 +145,14 @@
 
   SubQueryRemoveRule SUB_QUERY_JOIN_REMOVE_RULE =
       new SubQueryRemoveRule.SubQueryJoinRemoveRule(DrillRelFactories.LOGICAL_BUILDER);
+
+  FilterAggregateTransposeRule DRILL_FILTER_AGGREGATE_TRANSPOSE_RULE =
```

Comment:
Could you add this Instance in the `DrillFilterAggregateTransposeRule` as `DRILL_INSTANCE`? 
Do we want to match Filter and Aggregate with mixed `CONVENTION` `Trait` (see `DrillFilterAggregateTransposeRule#matches()`)?

Reply:
Done.